src/lib/y2storage/callbacks/activate.rb

```diff
@@ -56,9 +60,15 @@ def multipath(looks_like_real_multipath)
         )
       end
 
-      def luks(uuid, attempt)
-        log.info("Trying to open luks UUID: #{uuid} (#{attempt} attempts)")
-        dialog = Dialogs::Callbacks::ActivateLuks.new(uuid, attempt)
+      # Decides whether a LUKS device should be activated
+      #
+      # @param info [Storage::LuksInfo]
+      # @param attempt [Numeric]
+      #
+      # @return [Storage::PairBoolString]
+      def luks(info, attempt)
+        log.info("Trying to open luks UUID: #{info.uuid} (#{attempt} attempts)")
```

Comment:
NP: shall we uses `attempt(s)` instead?

```
Trying to open luks UUID: 2c8bc367-1058-46c5-8d3b-f8b75242dc97 (1 attempt(s))
```

Just asking, I have no strong opinion. Although I think we already discussed something similar in the past...

Reply:
Forget it, I just realize it is a `log` line.

Reply:
Yes, right now only the log informs about the number of attempts. In fact, nothing indicates to the user whether the password was wrong, but simply the popup is shown again. I think we can do it better when the user gives a wrong password:

a) explicitly telling to the user that the password was wrong and
b) informing about how many attempts have been performed.

@ancorgs what do you think?

Reply:
I agree

Reply:
> I agree

Me too